native/sapp-wasm/js/gl.js

```diff
@@ -1170,6 +1170,18 @@ var importObject = {
             } else {
                 document.exitPointerLock();
             }
+        },
+        sapp_set_fullscreen: function(fullscreen) {
+            if (!fullscreen) {
+                document.exitFullscreen();
+            } else {
+                canvas.requestFullscreen();
+            }
+        },
+        sapp_set_window_size: function(new_width, new_height) {
+            canvas.width = new_width;
+            canvas.height = new_height;
+            resize(canvas, wasm_exports.resize);
```

Comment:
does it really works? If I remember correctly resize will be called on the window resize, so it looks like set_window_size will work only until the next window resize JS event?

Reply:
Yah this works, I tested it out using the `index.html` in the repo and by removing `width: 100%, height: 100%`.